.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -136,9 +136,9 @@ body:
       options:
         - Docker
         - Docker Compose
-        - Podman
-        - Kubernetes
-        - Other
+        - Podman (not officially supported)
+        - Kubernetes (not officially supported)
+        - Other (unsupported)
```

Comment:
Same here. I assume you see value in having Podman and Kubernetes as separate options instead of a "Unsupported (Podman, Kubernetes, etc)"? I can understand why.

I know some maintainers have k8s experience; is that also the case with podman, or is it just common enough to provide some value here the report?

Reply:
Same as above :)

Reply:
Same as above :)